gradle/libs.versions.toml

```diff
@@ -40,7 +40,7 @@ compose-runtime = { module = "androidx.compose.runtime:runtime", version.ref = "
 compose-ui-test-junit4 = { module = "androidx.compose.ui:ui-test-junit4", version.ref = "compose" }
 compose-ui-tooling = { module = "androidx.compose.ui:ui-tooling", version.ref = "compose" }
 compose-ui-ui = { module = "androidx.compose.ui:ui", version.ref = "compose" }
-google-accompanist-flow = "com.google.accompanist:accompanist-flowlayout:0.20.0"
+google-accompanist-flow = "com.google.accompanist:accompanist-flowlayout:0.25.1"
```

Comment:
This is only used in our sandbox, so I don't think it's worth calling out in the changelog